src/Umbraco.Web.Common/Controllers/UmbracoPageController.cs

```diff
@@ -90,6 +90,13 @@ protected IActionResult CurrentTemplate<T>(T model)
         /// <param name="template">The view name.</param>
         protected bool EnsurePhsyicalViewExists(string template)
         {
+            if (string.IsNullOrEmpty(template))
```

Comment:
Could it be worth checking on IsNullOrWhitespace instead of IsNullOrEmpty?

Reply:
Good point since a template with just whitespace is not valid, I've updated it so we now use `IsNullOrWhiteSpace`